libs/projectcalico/config.jsonnet

```diff
@@ -0,0 +1,34 @@
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = ['3.15.0','3.16.0','3.17.0','3.18.0','3.19.0','3.20.0'];
+
+config.new(
+  name='projectcalico',
+  specs=[
+    {
+      output: version,
```

Comment:
We usually version by minor. So `3.15`, `3.16` since CRDs shouldn't change between patch versions